scripts/sql/dbsetup.sql

```diff
@@ -156,6 +192,14 @@ CREATE TABLE endpoint_organization (
     CONSTRAINT endpoint_org PRIMARY KEY (url, organization_npi_id)
 );
 
+CREATE TABLE fhir_endpoints_availability (
+    url             VARCHAR(500),
```

Comment:
do we always want to keep the availability, even if the the url no longer exists? my gut reaction is to delete the entry if the url is no longer in the `fhir_endpoints` table, but i can understand keeping it around to reference for historical purposes

Reply:
I asked Matt sometime ago about the scenario where if a endpoint was removed and added back again, should the availability of it include http statuses from before it was removed. And he had said yes if it's the same URL. So for that reason I didn't remove entries for removed URLs. But I could see the table getting really big if an endpoint was retired and not going to be in use again. @mayerm94 Thoughts?

Reply:
i mean that makes sense to me!